src/navigation/linking.ts

```diff
@@ -25,4 +27,73 @@ export const linking: LinkingOptions<NavigatorParamList> = {
       },
     },
   },
+  getStateFromPath(path, options) {
+    if (Object.keys(pathToNamedGameMode).includes(path)) {
```

Comment:
NIT - I think these are both equivalent in function but faster? (Not that speed matters here)
```suggestion
    if (pathToNamedGameMode[path]) {
```

```suggestion
    if (path in pathToNamedGameMode) {
```

Reply:
done